lib/license_finder/decisions.rb

```diff
@@ -281,6 +281,13 @@ def self.restore(persisted, result = new)
       return result unless persisted
 
       actions = YAML.load(persisted)
+
+      list_of_actions = (actions || []).map(&:first)
+
+      if (list_of_actions & %i{whitelist blacklist}).any?
+        raise 'The dependency_decisions.yml seems to have whitelist/blacklist keys which are deprecated. Please replace them with permit/restrict respectively and try again! More info - https://github.com/pivotal/LicenseFinder/commit/a40b22fda11b3a0efbb3c0a021381534bc998dd9'
```

Comment:
instead of calling it `dependency_decisions.yml`, i would use `decisions file` because users can pass it in as parameter as well (where filename could be different)

Reply:
changed 👍